src/python/Makefile

```diff
@@ -63,15 +63,15 @@ create-env:
 	@echo "-- Creating environment file."
 	[ -f foglamp/foglamp-env.yaml ] && echo "foglamp-env already exists." || cp foglamp/foglamp-env.example.yaml foglamp/foglamp-env.yaml
 
-test: py-test doc-test
+test: py-test doc-build-test
 
 py-test:
 	@echo "-- Executing python tests"
```

Comment:
Some @echo's end with periods and some do not. They should be consistent. I vote for no periods